bynder_sdk/util.py

```diff
@@ -11,6 +21,11 @@ def parse_json_for_response(response):
 
 class SessionMixin:
     def wrapped_request(self, func, endpoint, *args, **kwargs):
+        if 'headers' in kwargs:
+            kwargs['headers'].update(UA_HEADER)
+        else:
+            kwargs['headers'] = UA_HEADER
```

Comment:
Since we advertise 3.5 support I think we should stick with using format, raising it to 3.6 just for f-strings is not worth it in my opinion. We can reconsider this in the future of course.